packages/common/src/types/slackSettings.ts

```diff
@@ -9,6 +9,7 @@ export type SlackSettings = {
     notificationChannel: string | undefined;
     appProfilePhotoUrl: string | undefined;
     slackChannelProjectMappings?: SlackChannelProjectMapping[];
+    slackEnabled: boolean;
```

Comment:
Using 'installed' as it correlates to an organization installing the slack app. We have been using "enabled" terminology when enabling slack or other third party services for the Lightdash instance.

```suggestion
    isSlackInstalled: boolean;
```